platform/commonUI/edit/test/actions/SaveAsActionSpec.js

```diff
@@ -156,6 +156,28 @@ define(
                 expect(SaveAsAction.appliesTo(actionContext)).toBe(false);
             });
 
+            it("uses the editor capability to save the object", function () {
+                mockEditorCapability.save.andReturn(new Promise(function () {}));
```

Comment:
We tend to use [mock promises](https://github.com/BogdanAlexandru/openmct/blob/2a2e9ef99df364a9b9c2780178caff40c174833f/platform/commonUI/edit/test/actions/SaveAsActionSpec.js#L43) in situations like this. They are promises that are resolved immediately upon a call to `.then()`, which allows asynchronous code to be tested synchronously. I'd rather not add calls to `runs` and `waitsFor` as these have been deprecated in Jasmine 2.